cfpb/consumerfinance.gov
# TDP: Remember filters when a new search query is used

If you performed a new search term query with TDP, the new search query cleared out the existing filters. There was a request to retain the filters across query searches. This PR does that and adds a test for that type of search.

## Changes

- TDP: Remember filters when a new search query is used.
- Clear all filters now also clears the keyword query. 

## How to test this PR

1. `yarn build` and visit http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/ and perform a keyword search.
2. Apply one or more filters.
3. Change the keyword query and search again. The selected filters should remain.
4. Click "Clear all filters" and see that the filters and the search query are cleared.
